### PR TITLE
[SPARK-29168][WebUI][FOLLOW-UP] Use a dark colors on selected Executor removed on timeline view

### PR DESCRIPTION
**Pull request by TomokoKomiyama**
### What changes were proposed in this pull request?
Changed Executor color settings in timeline-view.css

### Why are the changes needed?
In WebUI, color of executor changes to dark blue when you click it. It might be confused user because of the color.

[ Before ]
r_before

[ After ]
r_after

### Does this PR introduce any user-facing change?
Yes.

### How was this patch tested?
Manually test.

### CODE REVIEW

**Comment by sarutak** [comment]
ok to test.

File: core/src/main/resources/org/apache/spark/ui/static/timeline-view.css [hunk 1]
```diff
@@ -207,6 +207,12 @@ rect.getting-result-time-proportion {
   border-color: #3EC0FF;
 }
 
+.vis-timeline .vis-item.executor.added.vis-selected {
+    background-color: #00AAFF;
+    border-color: #184C66;
+    z-index: 2;
+}
+
```
Comment (sarutak) [inline core/src/main/resources/org/apache/spark/ui/static/timeline-view.css]: Indentation should be fixed.

File: core/src/main/resources/org/apache/spark/ui/static/timeline-view.css [hunk 2]
```diff
@@ -217,17 +223,17 @@ rect.getting-result-time-proportion {
   border-color: #FF4D6D;
 }
 
+.vis-timeline .vis-item.executor.removed.vis-selected {
+    background-color: #FF6680;
+    border-color: #661F2C;
+    z-index: 2;
+}
+
```
Comment (sarutak) [inline core/src/main/resources/org/apache/spark/ui/static/timeline-view.css]: Same as the comment above.

**Review by sarutak — CHANGES_REQUESTED** [review]

File: core/src/main/resources/org/apache/spark/ui/static/timeline-view.css (hunk 1 above)
Comment (TomokoKomiyama) [reply core/src/main/resources/org/apache/spark/ui/static/timeline-view.css]: Thanks, I've done.

**Comment by TomokoKomiyama** [comment]
cc. @dongjoon-hyun

**Comment by sarutak** [comment]
retest this please.

**Review by sarutak — APPROVED** [review]

**Review by dongjoon-hyun — APPROVED** [review]

**Review by srowen — APPROVED** [review]

**Review by dongjoon-hyun — APPROVED** [review]
Merged to master. Thank you all!

**Pull request closed without merging**